### DISCUSSION TITLE
Welcome to awesome-github-profile-readme Discussions!

### DISCUSSION THREAD

**Discussion by abhisheknaiidu** [abhisheknaiidu/awesome-github-profile-readme#473] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sudip-mondal-2002** [comment]
Hi there, 
Can you look into my profile readme and consider it to be added in the list of awesome profiles?

**Comment by Villex-code** [comment]
Hello , could you by any chance select this for galaxy brain achievemnt ? You're awesome !

**Reply by sudip-mondal-2002** [reply]
Can you explain more? Hearing about it first time

**Comment by BadassBitch2022** [comment]
well, I have no idea what I'm doing or what coding is or even I'll GITHUB is used? I mean are there tutorials for a 40-year-old woman who is talk to use her phone by her young adult children?
